### PR TITLE
Prepare polygon release merged

### PR DESCRIPTION
**Pull request by vojin5rovic**

### CODE REVIEW

**Review by Gorshtak — APPROVED** [review]

File: src/pages/AMMTrading/components/AMM/AMM.tsx [hunk 1]
```diff
@@ -748,7 +805,7 @@ const AMM: React.FC = () => {
             <NetworkFees gasLimit={gasLimit} disabled={formDisabled} l1Fee={l1Fee} />
             {openApprovalModal && (
                 <ApprovalModal
-                    defaultAmount={sellAmount}
+                    defaultAmount={+(+sellAmount + 0.03 * +sellAmount).toFixed(2)}
```
Comment (vpoklopic) [inline src/pages/AMMTrading/components/AMM/AMM.tsx:808]: Why 0.03? Please put this as const.

File: src/pages/AMMTrading/components/Maturity/Maturity.tsx [hunk 2]
```diff
@@ -116,9 +117,20 @@ const Maturity: React.FC = () => {
...
-                gasLimit,
-            })) as ethers.ContractTransaction;
+
+            const gasPrice = await snxJSConnector.provider?.getGasPrice();
+            const gasInGwei = ethers.utils.formatUnits(gasPrice || 400000000000, 'gwei');
+
+            const providerOptions = isPolygon
+                ? {
+                      gasLimit,
+                      gasPrice: ethers.utils.parseUnits(Math.floor(+gasInGwei + +gasInGwei * 0.2).toString(), 'gwei'),
```
Comment (vpoklopic) [inline src/pages/AMMTrading/components/Maturity/Maturity.tsx:127]: Why 0.2? Please put this as const.

File: src/pages/Markets/Markets.tsx [hunk 3]
```diff
@@ -31,31 +32,35 @@ const Markets: React.FC = () => {
             return openOrdersQuery.data;
         }
     }, [openOrdersQuery]);
+
     const optionsMarkets = useMemo(() => {
         if (marketsQuery.isSuccess && Array.isArray(marketsQuery.data)) {
             const markets = openOrdersMap
                 ? marketsQuery.data.map((m) => {
                       const apiData = (openOrdersMap as any).get(m.address.toLowerCase());
+                      console.log(apiData);
```
Comment (vpoklopic) [inline src/pages/Markets/Markets.tsx:41]: Remove console.log.

File: src/pages/Markets/Markets.tsx [hunk 4]
```diff
@@ -31,31 +32,35 @@ const Markets: React.FC = () => {
...
                       return {
                           ...m,
                           openOrders: apiData?.ordersCount ?? 0,
                           availableLongs: apiData?.availableLongs ?? 0,
                           availableShorts: apiData?.availableShorts ?? 0,
-                          longPrice: apiData?.longPrice ?? 0,
-                          shortPrice: apiData?.shortPrice ?? 0,
+                          longPrice: +(networkId === POLYGON_ID ? apiData?.longPrice * 1e12 : apiData?.longPrice) ?? 0,
```
Comment (vpoklopic) [inline src/pages/Markets/Markets.tsx:47]: Why 1e12? Please put this as const.

**Review by makiBaraba — APPROVED** [review]

**Review by vpoklopic — APPROVED** [review]

**Pull request merged**